doc/source/tutorial/stats/continuous_kstwo.rst

```diff
@@ -0,0 +1,114 @@
+
+.. _continuous-kstwo:
+
+KStwo Distribution
+==================
+
+This is the distribution of the maximum absolute differences between an
+empirical distribution function, computed from :math:`n` samples or observations,
+and a comparison (or target) cumulative distribution function.
+(The "two" in the name is because this is the two-sided difference.
+``ksone`` is the distribution of the positive differences, :math:`D_n^+`,
+hence it concerns one-sided differences.
+``kstwobign`` is the limiting
+distribution of the *normalized* maximum absolute differences :math:`\sqrt{n} D_n`.)
+
+
+Writing :math:`D_n = \sup_t \left|F_{empirical,n}(t)-F_{target}(t)\right|`,
+``kstwo`` is the distribution of the :math:`D_n` values.
+
+
+``kstwo`` can also be used with the differences between two empirical distribution functions,
+for sets of observations with :math:`m` and :math:`n` samples respectively.
+Writing :math:`D_{m,n} = \sup_t \left|F_{1,m}(t)-F_{2,n}(t)\right|`,  where
+:math:`F_{1,m}` and :math:`F_{2,n}` are the two empirical distribution functions, then
+:math:`Pr(D_{m,n} \le x) \approx Pr(D_N \le x)` under appropriate conditions,
+where :math:`N = \sqrt{\left(\frac{mn}{m+n}\right)}`.
+
+
+There is one shape parameter :math:`n`, a positive integer, and the support is :math:`x\in\left[0,1\right]`.
+
+The implementation follows Simard & L'Ecuyer, which combines exact algorithms of Durbin and Pomeranz
+with asymptotic estimates of Li-Chien, Pelz and Good to compute the CDF with 5-15 accurate digits.
+
+Examples
+========
+
+>>> from scipy.stats import kstwo
+
+Show the probability of a gap at least as big as 0, 0.5 and 1.0 for a sample of size 5
+
+>>> kstwo.sf([0, 0.5, 1.0], 5)
+array([1.   , 0.112, 0.   ])
+
+Compare a sample of size 5 drawn from a source N(0.5, 1) distribution against
+a target N(0, 1) CDF.
+
+>>> from scipy.stats import norm
+>>> n = 5
+>>> gendist = norm(0.5, 1)       # Normal distribution, mean 0.5, stddev 1
+>>> np.random.seed(seed=233423)  # Set the seed for reproducibility
```

Comment:
if we use this seed, the chart does not look very nice since Dn+ is too small to be shown.

By trying a few more seeds, I noted that 423 looks better :)

Reply:
Changed to 423.